### DISCUSSION TITLE
Manual validation in Datagrid

### DISCUSSION THREAD

**Discussion by dockinkong** [Megabit/Blazorise#3497] [Q&A] [answered]
Hello,

Is it possible to use manual validation in Datagrid and if so where should I position Validations tag?...

I tried to put Validations tag as container of datagrid columns but individual Validation in evenry edit templates are not added to Validations...

Best regards

**Comment by stsrki** [comment]
DataGrid is already set to Manual mode. There is no way to change it at this time. Although, it might not be a bad idea to expose the `ValidationMode` to be able to change it.

**Reply by dockinkong** [reply]
But when I add Validations tag with a ref myModelValidations to contain the columns, individual validations are not added to myModelValidations... I can't control my edited model with myModelValidations.ValidateAll() and myModelValidations..ClearAll() as myModelValidations seems to not be cascaded to EditTemplate

**Comment by stsrki** [comment, marked as answer]
Validations component is already included in the DataGrid and it doesn't use the one outside. In your case, your `myModelValidations` would be ignored.

**Reply by dockinkong** [reply]
Ok thanks.
It could be interesting to get access to internal validations component or to specify it as property :)
Have a nice day
